### DISCUSSION TITLE
Enhancement: Saved presets can include attachments.

### DISCUSSION THREAD

**Discussion by Passerby1011** [danny-avila/LibreChat#3042] [Feature Requests & Suggestions]
### What features would you like to see added?

The current save preset can save the prompt. It is hoped that the feature of attaching files in the preset can be added.

### More details

By using the RAG function, including professional documentation in the presets can greatly enhance the capabilities of the dialogue assistant

### Which components are impacted by your request?

Presets, file management

### Pictures
![image](https://github.com/danny-avila/LibreChat/assets/63802758/2ca25366-8eb0-4ec2-81a0-549c4ee0f5d9)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
You will be able to pre-attach files via another mechanism in the future which may or may not involve presets

**Reply by beliven-valentino-cossar** [reply]
Hi @danny-avila is there a discussion thread or a pull request to monitor for this implementation?
